cpp/tests/column/column_view_test.cpp

```diff
@@ -54,23 +59,30 @@ struct ColumnViewAllTypesTests : public cudf::test::BaseFixture {
 TYPED_TEST_CASE(ColumnViewAllTypesTests, cudf::test::FixedWidthTypes);
 
 template <typename FromType, typename ToType, typename Iterator>
-void do_logical_cast(cudf::column_view const& input, Iterator begin, Iterator end)
+void do_logical_cast(cudf::test::detail::column_wrapper& input, Iterator begin, Iterator end)
```

Comment:
This should not be `column_wrapper`.

Reply:
I don't know how to change here. I used `column_wrapper` to adapt both column_view and mutable_column_view.

Reply:
`mutable_column_view` is implicitly convertible to `column_view`.

Reply:
Thanks, I've updated with implicit conversion.